apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/TrygdetidGrunnlagListe.tsx

```diff
@@ -0,0 +1,215 @@
+                <Table.HeaderCell scope={'col'}> </Table.HeaderCell>
+                <Table.HeaderCell scope={'col'}> </Table.HeaderCell>
+              </Table.Row>
+            </Table.Header>
+            <Table.Body>
+              {trygdetidGrunnlagListe.map((periode) => {
+                return (
+                  <PeriodeRow
+                    trygdetidGrunnlag={periode}
+                    behandlingId={trygdetid.behandlingId}
+                    setTrygdetid={setTrygdetid}
+                    endrePeriode={(trygdetidGrunnlagId) => {
+                      setEndreModus({ status: true, trygdetidGrunnlagId: trygdetidGrunnlagId })
+                    }}
+                    landListe={landListe}
+                    key={periode.id}
+                  />
+                )
+              })}
+            </Table.Body>
+          </Table>
+        </TableWrapper>
+      ) : (
+        <p>Ingen perioder registert</p>
+      )}
+
+      <NyEllerOppdatertPeriode>
+        {endreModus.status ? (
+          <TrygdetidGrunnlag
+            setTrygdetid={oppdaterStateOgSettTrygdetid}
+            avbryt={() => setEndreModus({ status: false, trygdetidGrunnlagId: '' })}
+            eksisterendeGrunnlag={trygdetidGrunnlagListe.find((tg) => tg.id == endreModus.trygdetidGrunnlagId)}
+            trygdetidGrunnlagType={trygdetidGrunnlagType}
+            landListe={landListe}
+          />
+        ) : (
+          <Button size="small" onClick={() => setEndreModus({ status: true, trygdetidGrunnlagId: '' })}>
+            Legg til ny periode
+          </Button>
+        )}
+      </NyEllerOppdatertPeriode>
+    </GrunnlagListe>
+  )
+}
+
+const PeriodeRow = ({
+  trygdetidGrunnlag,
+  behandlingId,
+  setTrygdetid,
+  endrePeriode,
+  landListe,
+}: {
+  trygdetidGrunnlag: ITrygdetidGrunnlag
+  behandlingId: string
+  setTrygdetid: (trygdetid: ITrygdetid) => void
+  endrePeriode: (trygdetidGrunnlagId: string) => void
+  landListe: ILand[]
+}) => {
+  const [slettTrygdetidStatus, slettTrygdetidsgrunnlagRequest] = useApiCall(slettTrygdetidsgrunnlag)
+
+  const slettGrunnlag = (grunnlagId: string) => {
+    slettTrygdetidsgrunnlagRequest(
+      {
+        behandlingsId: behandlingId,
+        trygdetidGrunnlagId: grunnlagId,
+      },
+      (oppdatertTrygdetid) => {
+        setTrygdetid(oppdatertTrygdetid)
+      }
+    )
+  }
+
+  const beregnetTrygdetid = trygdetidGrunnlag?.beregnet
+    ? `${trygdetidGrunnlag.beregnet.aar} år ${trygdetidGrunnlag.beregnet.maaneder} måneder ${trygdetidGrunnlag.beregnet.dager} dager`
+    : '-'
+
+  return (
+    <Table.ExpandableRow
+      expansionDisabled={!trygdetidGrunnlag.begrunnelse}
+      content={
+        <div>
+          <Heading size={'small'}>Begrunnelse</Heading>
+          {trygdetidGrunnlag.begrunnelse}
+        </div>
+      }
+    >
+      <Table.DataCell>
+        {landListe.find((land) => land.isoLandkode == trygdetidGrunnlag.bosted)?.beskrivelse?.tekst}
```

Comment:
Sikkert ikke viktig men stusser over at `trygdetidGrunnlag` kaller isoLandkode for "bosted" det rimer ikke helt.

Reply:
Ja, det kan godt være vi ønsker å endre navnet på det etterhvert. Tar det isåfall i en senere PR 👍